install-libs.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+#  Copyright 2023 Canonical Ltd.
+#  See LICENSE file for licensing details.
+
+# Script that installs the required native libraries for installing the plugin's dependencies
+# on Ubuntu 22.04.
+
+sudo apt update
```

Comment:
nitpick but we should avoid `apt` in scripts in favor of `apt-get`

Less nitpicky: I don't like the use of sudo here. The script should be executed as sudo if the user wants it. You can eventually test that you are root at the start of the script if you want to add a message and exit cleanly